### PR TITLE
editor imports plugin-commons styles

### PR DESCRIPTION
**Pull request by lxgreen**

### CODE REVIEW

File: examples/main/shared/editor/EditorPlugins.jsx [hunk 1]
```diff
@@ -51,7 +51,6 @@ import { createIndentPlugin } from 'wix-rich-content-plugin-indent';
 import 'wix-rich-content-editor-common/dist/styles.min.css';
 import 'wix-rich-content-common/dist/styles.min.css';
 import 'wix-rich-content-plugin-commons/dist/styles.min.css';
-import 'wix-rich-content-editor/dist/styles.min.css';
 import 'wix-rich-content-plugin-button/dist/styles.min.css';
```
Comment (oded-soffrin) [inline examples/main/shared/editor/EditorPlugins.jsx:54]: you mean to remove plugin-common, right?

File: packages/editor/web/src/index.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import RichContentEditor from './RichContentEditor';
 import RichContentEditorModal from './RichContentEditor/RichContentEditorModal';
+import 'wix-rich-content-plugin-commons/dist/styles.min.css';
```
Comment (oded-soffrin) [inline packages/editor/web/src/index.ts:3]: not needed  IMO

File: packages/editor/web/statics/styles/rich-content-editor.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
 @import '~wix-rich-content-common/dist/statics/styles/consts';
 @import '~wix-rich-content-editor-common/dist/statics/styles/general';
 @import '~wix-rich-content-common/dist/statics/styles/fonts';
+@import 'rich-content-plugins-common/dist/statics/styles.min.css';
```
Comment (oded-soffrin) [inline packages/editor/web/statics/styles/rich-content-editor.scss:5]: @import '~wix-rich-content-plugin-commons/distt/statics/styles.min.css`;

File: examples/main/shared/editor/EditorPlugins.jsx (hunk 1 above)
Comment (lxgreen) [reply examples/main/shared/editor/EditorPlugins.jsx:54]: Yep :facepalm:

File: packages/editor/web/statics/styles/rich-content-editor.scss (hunk 3 above)
Comment (lxgreen) [reply packages/editor/web/statics/styles/rich-content-editor.scss:5]: Doh.. need to sleep more

File: packages/editor/web/src/index.ts (hunk 2 above)
Comment (lxgreen) [reply packages/editor/web/src/index.ts:3]: Yet another attempt

**Pull request merged**